api/v1/cluster_webhook.go

```diff
@@ -1009,6 +1028,17 @@ func (r *Cluster) validateConfiguration() field.ErrorList {
 		}
 	}
 
+	if value := r.Spec.PostgresConfiguration.Parameters[sharedBuffersParameter]; value != "" {
+		if _, err := parsePostgresQuantityValue(value); err != nil {
```

Comment:
the error is hidden, it is better to let the user know what went wrong,  or what is the correct format

Reply:
Updated the message, is it any better?